src/browser.ts

```diff
@@ -71,6 +84,12 @@ export class GrpcClient {
     this.promise = 'promise' in options ? options.promise! : Promise;
   }
 
+  /**
+   * Browser version of loadProto
+   * Loads the gRPC service from a JSON object created from a proto file
```

Comment:
I think that loadProto() actually loads a protobuf root, from which you can extract the the protobuf service afterwards